doc/en/how-to/cache.rst

```diff
@@ -185,6 +185,9 @@ using the ``--last-failed-no-failures`` option, which takes one of the following
     pytest --last-failed --last-failed-no-failures all    # run all tests (default behavior)
     pytest --last-failed --last-failed-no-failures none   # run no tests and exit
 
+The ``--last-failed-no-failures`` option governs the behavior of ``--if``.
+Default ``all`` runs all tests with no known failures. ``none`` avoids tests and exits successfully when there are no known failures.
```

Comment:
IMHO this does not add much to what we already have above (and people found lacking in the first place).

Feel free to not only add to this section, but rewrite it entirely if you think would be

Reply:
update the docs here: 75e815f6c1dd5705c3a1f6d11b320013775b59e4